apps/box/src/components/Cards/ConnectedDevicesCard.tsx

```diff
@@ -53,29 +52,21 @@ const DeviceCard = ({
           </FxTag>
         ))}
       </FxBox>
-      <FxBox flexDirection="row" justifyContent="space-between">
-        <FxText color="content1" variant="bodySmallRegular">
-          Capacity
-        </FxText>
-        <FxText color="content2" variant="bodySmallLight">
-          {convertMegabyteToGigabyte(capacity)} GB
-        </FxText>
-      </FxBox>
-      <FxHorizontalRule marginVertical="12" />
-      <FxBox flexDirection="row" justifyContent="space-between">
-        <FxText color="content1" variant="bodySmallRegular">
-          Status
-        </FxText>
+      <CardRow>
+        <CardRowTitle>Capacity</CardRowTitle>
+        <CardRowData>{convertMegabyteToGigabyte(capacity)} GB</CardRowData>
+      </CardRow>
+      <CardRow>
+        <CardRowTitle>Status</CardRowTitle>
         <FxBox flexDirection="row" alignItems="center">
-          <FxText color="content2" variant="bodySmallLight">
+          <CardRowData>
```

Comment:
Does it make sense to break thing down to this granular level? Unless `<CardRowData>` is going to be used multiple times.

Reply:
In the UI Library there are 4 or 5 cards that have similar Row, title, data, formatting that these can be used on.